From 8aa079588da39bcbcaa25d353fd0ede028775b03 Mon Sep 17 00:00:00 2001
From: emi <emiliorizzo@gmail.com>
Date: Thu, 9 Sep 2021 20:22:27 -0300
Subject: [PATCH] Update platformio.ini

---
 platformio.ini | 32 +++++++++++++++-----------------
 1 file changed, 15 insertions(+), 17 deletions(-)

diff --git a/platformio.ini b/platformio.ini
index f55f5f5a93..35cfd5edf8 100644
--- a/platformio.ini
+++ b/platformio.ini
@@ -10,10 +10,14 @@
 # Automatic targets - enable auto-uploading
 #targets = upload
 
+#BIQU-B1 Printer
+#  default_envs = LPC1768
+#  default_envs = BIGTREE_SKR_2
+
 [platformio]
 src_dir      = Marlin
 boards_dir   = buildroot/share/PlatformIO/boards
-default_envs = mega2560
+default_envs = BIGTREE_SKR_2
 include_dir  = Marlin
 extra_configs =
     ini/avr.ini
@@ -24,7 +28,6 @@ extra_configs =
     ini/native.ini
     ini/samd51.ini
     ini/stm32f0.ini
-    ini/stm32f1-maple.ini
     ini/stm32f1.ini
     ini/stm32f4.ini
     ini/stm32f7.ini
@@ -37,7 +40,7 @@ extra_configs =
 # Remove '-fmax-errors=5' from build_flags below to see all.
 #
 [common]
-build_flags        = -g3 -D__MARLIN_FIRMWARE__ -DNDEBUG
+build_flags        = -g3 -D__MARLIN_FIRMWARE__ -DNDEBUG -fmerge-constants
                      -fmax-errors=5
 extra_scripts      =
   pre:buildroot/share/PlatformIO/scripts/common-dependencies.py
@@ -71,14 +74,15 @@ default_src_filter = +<src/*> -<src/config> -<src/HAL> +<src/HAL/shared>
   -<src/lcd/menu/menu_touch_screen.cpp>
   -<src/lcd/menu/menu_tramming.cpp>
   -<src/lcd/menu/menu_ubl.cpp>
-  -<src/lcd/extui/anycubic_chiron>
-  -<src/lcd/extui/anycubic_i3mega>
-  -<src/lcd/extui/dgus> -<src/lcd/extui/dgus/fysetc> -<src/lcd/extui/dgus/hiprecy> -<src/lcd/extui/dgus/mks> -<src/lcd/extui/dgus/origin>
-  -<src/lcd/extui/example>
-  -<src/lcd/extui/ftdi_eve_touch_ui>
-  -<src/lcd/extui/malyan>
-  -<src/lcd/extui/mks_ui>
-  -<src/lcd/extui/nextion>
+  -<src/lcd/extui/lib/mks_ui>
+  -<src/lcd/extui/lib/dgus> -<src/lcd/extui/dgus_lcd.cpp>
+  -<src/lcd/extui/lib/dgus/fysetc> -<src/lcd/extui/lib/dgus/hiprecy> -<src/lcd/extui/lib/dgus/mks> -<src/lcd/extui/lib/dgus/origin>
+  -<src/lcd/extui/example.cpp>
+  -<src/lcd/extui/malyan_lcd.cpp>
+  -<src/lcd/extui/lib/ftdi_eve_touch_ui>
+  -<src/lcd/extui/anycubic_chiron_lcd.cpp> -<src/lcd/extui/lib/anycubic_chiron>
+  -<src/lcd/extui/anycubic_i3mega_lcd.cpp> -<src/lcd/extui/lib/anycubic_i3mega>
+  -<src/lcd/extui/nextion_lcd.cpp> -<src/lcd/extui/lib/nextion>
   -<src/lcd/lcdprint.cpp>
   -<src/lcd/touch/touch_buttons.cpp>
   -<src/sd/usb_flashdrive/lib-uhs2> -<src/sd/usb_flashdrive/lib-uhs3>
@@ -156,7 +160,6 @@ default_src_filter = +<src/*> -<src/config> -<src/HAL> +<src/HAL/shared>
   -<src/gcode/calibrate/M425.cpp>
   -<src/gcode/calibrate/M666.cpp>
   -<src/gcode/calibrate/M852.cpp>
-  -<src/gcode/control/M10-M11.cpp>
   -<src/gcode/control/M42.cpp> -<src/gcode/control/M226.cpp>
   -<src/gcode/config/M43.cpp>
   -<src/gcode/config/M217.cpp>
@@ -199,14 +202,10 @@ default_src_filter = +<src/*> -<src/config> -<src/HAL> +<src/HAL/shared>
   -<src/gcode/geometry/M206_M428.cpp>
   -<src/gcode/host/M16.cpp>
   -<src/gcode/host/M113.cpp>
-  -<src/gcode/host/M154.cpp>
   -<src/gcode/host/M360.cpp>
   -<src/gcode/host/M876.cpp>
   -<src/gcode/lcd/M0_M1.cpp>
-  -<src/gcode/lcd/M117.cpp>
   -<src/gcode/lcd/M250.cpp>
-  -<src/gcode/lcd/M300.cpp>
-  -<src/gcode/lcd/M414.cpp>
   -<src/gcode/lcd/M73.cpp>
   -<src/gcode/lcd/M995.cpp>
   -<src/gcode/motion/G2_G3.cpp>
@@ -226,7 +225,6 @@ default_src_filter = +<src/*> -<src/config> -<src/HAL> +<src/HAL/shared>
   -<src/gcode/temp/M104_M109.cpp>
   -<src/gcode/temp/M155.cpp>
   -<src/gcode/units/G20_G21.cpp>
-  -<src/gcode/units/M82_M83.cpp>
   -<src/gcode/units/M149.cpp>
   -<src/libs/BL24CXX.cpp> -<src/libs/W25Qxx.cpp>
   -<src/libs/L64XX> -<src/module/stepper/L64xx.cpp> -<src/HAL/shared/HAL_spi_L6470.cpp>
-- 
2.25.1

